src/tsoder_app.erl

```diff
@@ -14,7 +14,19 @@
 %% API
 %%====================================================================
 
+log_file_name(Suffix) ->
+    string:join(["log_", Suffix, ".txt"], "").
```

Comment:
According to [the manual](http://erlang.org/doc/man/string.html), this function is obsolete and should be replaced with `lists:join`.